filament/src/MaterialInstance.cpp

```diff
@@ -226,6 +227,10 @@ Material const* MaterialInstance::getMaterial() const noexcept {
     return upcast(this)->mMaterial;
 }
 
+const char* MaterialInstance::getName() const noexcept {
+    return upcast(this)->mName.c_str();
```

Comment:
this should call `FMaterialInstance::getName()` instead of using `mName `directly

Reply:
Done.